.github/workflows/ci.yml

```diff
@@ -59,6 +50,29 @@ jobs:
         with:
           port: 6000
 
+      - name: Install and start MinIO
+        uses: addnab/docker-run-action@v3
+        with:
+          image: minio/minio:RELEASE.2025-07-23T15-54-02Z
+          options: --detach -p 9000:9000 -p 9001:9001 --env MINIO_ROOT_USER=root --env MINIO_ROOT_PASSWORD=tembatemba --volume ${{ github.workspace }}/data:/data
+          run: minio server /data --console-address ":9001"
+
+      - name: Install MinIO client
+        run: |
+          # Download MinIO client and its checksum
+          sudo curl -O https://dl.min.io/client/mc/release/linux-amd64/mc.RELEASE.2025-07-21T05-28-08Z
+          sudo curl -O https://dl.min.io/client/mc/release/linux-amd64/mc.RELEASE.2025-07-21T05-28-08Z.sha256sum
+          # Verify checksum
+          sha256sum -c mc.RELEASE.2025-07-21T05-28-08Z.sha256sum
+          sudo chmod +x mc.RELEASE.2025-07-21T05-28-08Z
+          sudo mv mc.RELEASE.2025-07-21T05-28-08Z /usr/local/bin/mc
+          mc alias set local http://127.0.0.1:9000 root tembatemba
+
+          for bucket in test-attachments test-logs test-sessions; do
+            mc ls local/$bucket || mc mb local/$bucket
+          done
+          mc anonymous set public local/test-attachments
```

Comment:
do we need the client? am looking at https://www.liquibase.com/blog/engineering-efficiency-at-liquibase-improving-aws-s3-integration-tests-with-minio#phase-2-github-actions-linux-runners

Reply:
We need the client to create the buckets. the previous image allowed passing a list of buckets and it created those for us